src/pages/resultsView/ResultsViewPage.tsx

```diff
@@ -223,6 +225,23 @@ export default class ResultsViewPage extends React.Component<IResultsViewPagePro
                 }
             },
 
+            {
+                id:ResultsViewTab.FUSION,
+                hide:()=>{
```

Comment:
!this.resultsViewPageStore.fusions.isComplete || this.resultsViewPageStore.fusions.result.length === 0